cfgov/unprocessed/apps/form-explainer/css/form-explainer.scss

```
@@ -103,38 +123,39 @@
     }
 
     .o-expandable__label {
-      font-size: unit(16 / @base-font-size-px, em);
+      font-size: 1em;
     }
 
-    each( @colors, {
-      @tint: @{value}-bg;
-      @light: @{value}-lt;
-      @med: @{value}-md;
+    @each $value in $colors {
+      $index: list.index($colors, $value);
+      $tint: #{$value}-bg;
+      $light: #{$value}-lt;
+      $med: #{$value}-md;
```

Needs to read from colors map:

```suggestion
      $tint: map.get($colors-map, #{$value}-bg);
      $light: map.get($colors-map, #{$value}-lt);
      $med: map.get($colors-map, #{$value}-md);
```

---

Thank you!